pycrdt_websocket/ystore.py

```diff
@@ -356,10 +366,11 @@ async def start(
                 self._task_group.start_soon(self._init_db)
                 task_status.started()
                 self.started.set()
+                await self.stopped.wait()
```

Comment:
Why do we need to make the start method blocking here? In websocketServer, we have used `self._task_group.start_soon(self._stopped.wait)` to keep the task group alive. Should we use this pattern?

Reply:
It's equivalent, in WebsocketServer the task group context manager doesn't exit as long as tasks are running.